### ISSUE TITLE
Customer Feedback Received

### ISSUE THREAD

**Issue by crichID** [certify/curriculum-github#184]
> I’m a first year undergraduate student, pursuing a Btech in Computer Science at IIIT Hyderabad, India. I’ve had almost negligible interaction with Git or GitHub (I know the difference now, yay!) earlier, and as such was a complete noob when it came to version control. I undertook the course knowing that I would need the knowledge in the future for projects at my university and perhaps even outside of my curriculum, and also because the course seemed quite convenient (not to mention free) and sated my curiosity regarding the tech.
> 
> It took me about three weeks to complete the course, in between which I had only one minor incidence where I by mistake deleted the main repository (github-for-developers-7). Due to this I was unable to move forward with the course. I sent wheelhouse (help@wheelhouse.io) an email but never got back a reply, despite waiting for a week. I then proceeded to forward the same email to training@github.com and was pleased to see a prompt reply in two days; whence Cynthia Rich, of services@github.com, (thanks Cynthia!) reinitialized the repository for me, and I was able to get on with the course.
> 
> The rest of the course went smoothly; it was a fun and inviting experience, which may be because I’m already well familiar with the Linux terminal and its usage, and I did not find the course to be too difficult. I was also quite pleased with the amount of material that the course covered. I would gladly recommend the course to anybody who wants to learn more regarding the subject.
> 
> TL;DR Loved the course; was disappointed by Wheelhouse’s customer support.

**Comment by 1point618** [comment]
What is the user's email? I'll try to figure out why we never got the email from them.

**Comment by crichID** [comment]
@1point618 her email is: adi.srivastava@hotmail.com

**Comment by 1point618** [comment]
Thank you!

**Comment by PeterBell** [comment]
@crichID we're also checking what happened with the email . . .

**Comment by 1point618** [comment]
@crichID Just to let you know, we figured out the problem that was causing email not to forward to intercom. We're getting everything from the email now.
